### DISCUSSION TITLE
I can't use my pre transcript as a form of proof

### DISCUSSION THREAD

**Discussion by katglang-student** [community/community#29687] [GitHub Education]
I've been trying to apply for the student developer pack and for some reason, it insists I submit a dated school ID despite me providing my transcript image. (picture below as proof)

Since my university classes have been remote, my university did not issue a physical school ID and the closest thing to a school ID I have doesn't even have a date i.e. it's actually a QR code issued by my school that has my matric number.

I tried to get a ticket for my issue but ironically, I have to be verified as a student which is exact the issue I'm having right now.

![IMG_20220816_214058_549](https://user-images.githubusercontent.com/111364852/184893917-82146af8-e007-4308-95ae-c4ff8a566857.png)

**Comment by TheAwiteb** [comment]
Same issue, i think there's a bug in image validation

> https://github.com/orgs/community/discussions/29656
